halo2_proofs/src/plonk/circuit.rs

```diff
@@ -45,6 +47,26 @@ impl<C: ColumnType> Column<C> {
     pub fn column_type(&self) -> &C {
         &self.column_type
     }
+
+    /// Return expression from column at a relative position
+    pub fn query_cell<F: Field>(&self, at: Rotation) -> Expression<F> {
+        self.column_type.query_cell(self.index, at)
+    }
+
+    /// Return expression from column at the current row
+    pub fn expr<F: Field>(&self) -> Expression<F> {
```

Comment:
Perhaps cur is a more clear name for this method?

Reply:
Yeah probably a good idea to at least also expose it as an option. I like the option for `expr` as well because a lot of other data types already use that to convert something to an expression.

Reply:
It is not obvious what a column.expr() should return, what does it mean to convert a column to a expression, in how many ways can be that interpreted? This method is doing a current row rotation and that is what it should say in my opinion

Reply:
Also, we have a next and prev method, it is obvious in my opinion that the cur one is missing.

Reply:
Should I add both `expr()` and `cur()` so that they can be used interchangeably?

Reply:
my opinion is to have only cur() for the current rotation. if there is another way to convert a column to an expression that it does not include a rotation, my opinion is that in that case expr could be useful but I cannot think of any rotation-less expression from a column